Submitty/Submitty
# [UI/UX:RainbowGrades] Move grade summaries to customization

### Please check if the PR fulfills these requirements:


Steps to Test
If on main:
1. Login to Instructor, then navigate to Grade Reports.
2. Click **Generate Grade Summaries** button

If on branch:
1. Login to Instructor, then navigate to Grade Reports > Web-Based Rainbow Grades Customization.
2. Click **Generate Grade Summaries** button
3. Button should have same functionality as on main 

### What is the current behavior?

The **Generate Grade Summaries** button is on the _Grade Reports_ page, which is in the process of being removed.

### What is the new behavior?
The **Generate Grade Summaries** button has been moved to the _Web-Based Rainbow Grades Generation_ page.
![image](https://github.com/user-attachments/assets/f48d6941-5e04-44b2-857f-a659edd79cab)
![image](https://github.com/user-attachments/assets/2f30d402-2cb9-4ed9-a679-bea0a28de29a)

### Other information?
